### ISSUE TITLE
Can't make plugin work with mkdocs-material Docker image

### ISSUE THREAD

**Issue by ericdallo** [Guts/mkdocs-rss-plugin#8]
Hi, we from `https://emacs-lsp.githb.io/lsp-mode` intend to use your plugin https://github.com/emacs-lsp/lsp-mode/issues/2184.
We use`mkdocs-material` which already works for us for months but when we add your plugin It seems to happen some error:

Error
![image](https://user-images.githubusercontent.com/7820865/93375089-fec8a300-f82d-11ea-9dc8-6c0094bc5904.png)

Docker image:
```Dockerfile
FROM squidfunk/mkdocs-material
RUN pip install --no-cache-dir \
        'mkdocs-awesome-pages-plugin>=2.2.1' \
        'mkdocs-git-revision-date-localized-plugin>=0.4' \
        'mkdocs-minify-plugin>=0.3' \
        'mkdocs-redirects>=1.0' \
        'mkdocs-rss-plugin>=0.6.1';
```

**Comment by ericdallo** [comment]
@Guts

**Comment by Guts** [comment]
Hi. Thanks for trying this plugin and taking time to report.

I'm already using this plugin with mkdocs-material:

- website: https://static.geotribu.fr
- feed of created content: https://static.geotribu.fr/feed_rss_created.xml
- feed of updated content: https://static.geotribu.fr/feed_rss_updated.xml

I'll have a look.

**Comment by Guts** [comment]
Quick test with Dockerfile:

```docker
FROM squidfunk/mkdocs-material
RUN pip install --no-cache-dir \
        'mkdocs-awesome-pages-plugin>=2.2.1' \
        'mkdocs-git-revision-date-localized-plugin>=0.4' \
        'mkdocs-minify-plugin>=0.3' \
        'mkdocs-redirects>=1.0' \
        'mkdocs-rss-plugin>=0.6.1';
```

Run: `docker build --pull --rm -f "Dockerfile" -t plugin-with-material:latest "."`

And this worked.

**Comment by Guts** [comment]
@ericdallo I've added a GA testing against docker build. Last run: https://github.com/Guts/mkdocs-rss-plugin/runs/1130269359

I've also checked if the method `get_build_timestamp` spotted into your log is still alive in mkdocs projects and it's okay. See https://github.com/mkdocs/mkdocs/blob/fa5aa4a26efc2a0dc3878b41920eaa39afc8929b/mkdocs/utils/__init__.py#L83-L94 and `from mkdocs.utils import get_build_timestamp` works fine.

Could you provide your mkdocs version please?

**Comment by ericdallo** [comment]
Thanks for the help!
I'm using the docker image here: https://github.com/emacs-lsp/lsp-mode/blob/master/.github/workflows/docs.yml#L34 so I think it's the latest.

**Comment by Guts** [comment]
> I'm using the docker image here: https://github.com/emacs-lsp/lsp-mode/blob/master/.github/workflows/docs.yml#L34 so I think it's the latest.

The shared line is only about the mkdocs-material image, isn't? `docker run --rm -v ${PWD}:/docs squidfunk/mkdocs-material -- build`. Am I missing some information?

**Comment by Guts** [comment]
FYI, as mentioned by @squidfunk in [this doc](https://squidfunk.github.io/mkdocs-material/deprecations/#bundled-plugins), he'll be following the Docker recomendations removing plugins you're actually using.

**Comment by ericdallo** [comment]
@Guts, I know I opened this issue right after closing [this](https://github.com/squidfunk/mkdocs-material/pull/1928), and I have this issue on my custom docker image that I did like this:

```dockerfile
FROM squidfunk/mkdocs-material
RUN pip install --no-cache-dir \
        'mkdocs-awesome-pages-plugin>=2.2.1' \
        'mkdocs-git-revision-date-localized-plugin>=0.4' \
        'mkdocs-minify-plugin>=0.3' \
        'mkdocs-redirects>=1.0' \
        'mkdocs-rss-plugin>=0.6.1';
```

**Comment by Guts** [comment]
@ericdallo I'm asking beacuse on my side, it's working:

![Capture d’écran de 2020-09-19 00-26-20](https://user-images.githubusercontent.com/1596222/93650470-dcda4680-fa0e-11ea-8207-acaa210f6539.png)

Steps:

1. cloned your repository
2. build a docker image with the bundled plugin with https://github.com/Guts/mkdocs-rss-plugin/blob/master/.github/workflows/docker-test.yml#L23 (tag = mkdocs-plugins/rss)
3. add `rss` to the mkdocs.yml file
4. launched the build with your command ` docker run --rm -v ${PWD}:/docs mkdocs-plugins/rss -- build`

RSS files are created (see in the screen capture).

**Comment by ericdallo** [comment]
Odd :thinking: , let me try again

**Comment by ericdallo** [comment]
Well, this still happens:
![image](https://user-images.githubusercontent.com/7820865/93651382-a937f600-f9e7-11ea-913d-9772bee6771a.png)

dokcer version: 19.03.12

Maybe some issue with my docker, I'll try to build the lsp-mode via Github action and see if it works

**Comment by ericdallo** [comment]
It actually worked on CI :D
Thanks for the help and your time @Guts
